tasks/db_deploy/migrations/migrate_db.py

```diff
@@ -5,8 +5,9 @@
 """
 from typing import Dict
 from orca_shared.database.shared_db import get_admin_connection, logger
-import migrate_db_v2, migrate_db_v3, migrate_db_v4
-
+from migrations.migrate_versions_1_to_2.migrate_db_v2 import migrate_versions_1_to_2
```

Comment:
Feels like you're missing a bit from the import, but I might be wrong. Otherwise looks good.
`migrations.migrate_versions_<x>_to_<y>.migration.migrate as migrate_versions_<x>_to_<y>`?

Reply:
I thought having same name `migration.py` and `migration_sql.py` under every folder will be a little confusing while working which is why I gave different names to the files. 

```
mkdir -p migrations/migrate_versions_<x>_to_<y>
touch migrations/migrate_versions_<x>_to_<y>/migrate_db_v2.py
touch migrations/migrate_versions_<x>_to_<y>/migrate_sql_v2.py
```

I think keeping the file names different will be easier to work with.

Thoughts? @AndrewDorn  @bhazuka

Reply:
I can see the argument for that. Does mean that we can't fully split the sql from the business logic.

Reply:
changed to migrate.py as per discussion